### DISCUSSION TITLE
Jkbms: Connection with RS485 AND Bluetooth possible?

### DISCUSSION THREAD

**Discussion by MisterX1000** [Louisvdw/dbus-serialbattery#751] [Q&A] [answered]
Hi,
In my system I use one Jkbms which is connected to my Victron MP2-GX via RS485 cable connection. Everything runs fine this way.

I was just wondering if is it allowed or forbidden to "activate" also the Bluetooth connection by entering the bms address in the config file of serialbattery?

- Will this work without problems? Or is it forbidden to connect the same bms in to ways? 
- When the RS485 cable is disconnected, will the serialbattery driver switch automatically from cable to BLE?

Thank you in advance for your answers and best regards
Alex

**Comment by mr-manuel** [comment, marked as answer]
You can do it, but does not make much sense. Why should the cable be unplugged? In this case it would be to place the cable better.
Theoretically the system should choose automatically the BMS available, so it would act like a failover. Anyway, I would not recommend it.

You can connect one GX device via serial and BLE or two different GX devices, one via serial the other via BLE.

**Reply by MisterX1000** [reply]
Thank you for your statement.

**Comment by Zaphod-dev** [comment]
Hi,
May I ask from what ports on the JK BMS and the MP2 are connected, and what type of cable you are using ?
Many thanks for your help and advice !
Pascal
